fakturogen-comarch-connector/src/main/java/pl/fakturogen/comarch/connector/connectors/ComarchApiTokenConnector.java

```diff
@@ -1,37 +1,40 @@
 package pl.fakturogen.comarch.connector.connectors;
 
-import okhttp3.*;
-import org.springframework.stereotype.Component;
+import okhttp3.Call;
+import okhttp3.FormBody;
+import okhttp3.Headers;
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.RequestBody;
+import okhttp3.Response;
+import okhttp3.ResponseBody;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+import pl.fakturogen.api.token.ApiToken;
+import pl.fakturogen.comarchconnector.converter.TokenResponseConverter;
 
 import java.io.IOException;
 import java.util.Base64;
 import java.util.HashMap;
 import java.util.Map;
 
-@Component
+@Service
+//@Setter
+// @PropertySource("classpath:credentials.properties")
 public class ComarchApiTokenConnector {
-
+    @Value("${comarch.credentials.clientId}")
     private String clientId;
+    @Value("${comarch.credentials.secret}")
     private String secret;
     private String url = "https://app.erpxt.pl/api2/public/token";
 
-    public ComarchApiTokenConnector() {
-    }
-
-    public ComarchApiTokenConnector(String clientId, String secret) {
-        this.clientId = clientId;
-        this.secret = secret;
-    }
-
-    public void setClientId(String clientId) {
-        this.clientId = clientId;
-    }
+    private TokenResponseConverter tokenResponseConverter;
 
-    public void setSecret(String secret) {
-        this.secret = secret;
+    public ComarchApiTokenConnector(TokenResponseConverter tokenResponseConverter) {
```

Comment:
please add setters for Secret and ClientId, as when we will have many users they will need to set their credentials